### PR TITLE
changelog: update changelog for 8.9 release

### PR DESCRIPTION
**Pull request by kruskall**
## Checklist

- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)
- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)
- [ ] Documentation has been updated

For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

### CODE REVIEW

File: docs/release-notes.asciidoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ This section summarizes the changes in each release.
 
 **APM integration and APM Server**
 
+* <<release-notes-8.9>>
```
Comment (bmorelli25) [inline docs/release-notes.asciidoc:12]: 😍

File: changelogs/8.9.asciidoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+[float]
+[[release-notes-8.9.0]]
+=== APM version 8.9.0
+
+[float]
+==== Breaking Changes
+
+[float]
+==== Deprecations
```
Comment (bmorelli25) [inline changelogs/8.9.asciidoc:16]: Are we expecting additions to these sections? If not, let's remove them.

**Review by bmorelli25 — APPROVED** [review]
Thanks! Just one suggestion.

File: changelogs/8.9.asciidoc (hunk 2 above)
Comment (kruskall) [reply changelogs/8.9.asciidoc]: Done! Thanks!

**Review by dmathieu — APPROVED** [review]

**Review by simitt — APPROVED** [review]

**Pull request merged**